web/packages/core/src/load-options.ts

```diff
@@ -300,6 +300,13 @@ export interface BaseLoadOptions {
      */
     forceScale?: boolean;
 
+    /**
+     * Sets the player's frame rate, overriding the frame rate set by the movie.
```

Comment:
Can we make this say "Sets and locks ..." just to make it clear this makes it static and not dynamic?